.jenkins/common.groovy

```diff
@@ -31,12 +31,22 @@ def runTestCommand (platform, project)
     String sudo = auxiliary.sudo(platform.jenkinsLabel)
     String centos = platform.jenkinsLabel.contains('centos') ? '3' : ''
     def testCommand = "ctest${centos} --output-on-failure"
+    def hmmTestCommand = ''
+    if (platform.jenkinsLabel.contains('gfx90a'))
+    {
+        hmmTestCommand = """
+                            export HSA_XNACK=1
+                            export ROCRAND_USE_HMM=1
+                            ${testCommand}
+                         """
+    }
 
     def command = """#!/usr/bin/env bash
                 set -x
                 cd ${project.paths.project_build_prefix}/build/release
                 make -j4
-                ${sudo} LD_LIBRARY_PATH=/opt/rocm/lib/ ${testCommand}
+                ${testCommand}
+                ${hmmTestCommand}
```

Comment:
`testCommand` is included in `hmmTestCommand` already.  Is intentional to include both in `command`?

Reply:
Yes, the idea is to run the full suite of tests on all gfx architectures. On gfx90a, we would repeat the suite but with hmm and xnack enabled.